### ISSUE TITLE
unit/regression test failure on mac osx 10.9.3 with hdf5 table creation

### ISSUE THREAD

**Issue by gidden** [cyclus/cyclus#981]
Unit test failures: https://gist.github.com/gidden/3143f27cd876c77a2357. Regression test failures: https://gist.github.com/gidden/18ad78ff657c59a369bd.

Our listserv discussion is here https://groups.google.com/forum/#!topic/cyclus-dev/7zZHaY7BaxE.

Through some of my own tinkering, I was able to determine that a failure is caused upon table creation in `Hdf5BackTests.ReadWriteVLMapVLStringDouble`, so we can start there.

**Comment by gidden** [comment]
I just saw these on Ubuntu, but only once, and I can't reproduce =/

**Comment by gidden** [comment]
closing this as we are implementing sql support for macs

**Comment by scopatz** [comment]
This is for HDF5 unit tests on macs, so this should remain open until they are actually fixed.

**Comment by gidden** [comment]
moving to 1.4, @mbmcgarry, can you comment as to whether you see these failures, please?

**Comment by mbmcgarry** [comment]
Yes, I still see errors with HDF5 in some of the the unit tests.

[  FAILED  ] 9 tests, listed below:
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapVLStringDouble
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapStringVLString
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapVLStringString
[  FAILED  ] Hdf5BackTests.ReadWriteMapVLStringVLString
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapVLStringVLString
[  FAILED  ] Hdf5BackTests.ReadWriteMapPairIntStringDouble
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapPairIntStringDouble
[  FAILED  ] Hdf5BackTests.ReadWriteMapPairIntVLStringDouble
[  FAILED  ] Hdf5BackTests.ReadWriteVLMapPairIntVLStringDouble

There are many other HDF5BackTests that do run successfully however.

-Meghan


On Apr 25, 2015, at 9:28 AM, Matthew Gidden <notifications@github.com<mailto:notifications@github.com>> wrote:


moving to 1.4, @mbmcgarry<https://github.com/mbmcgarry>, can you comment as to whether you see these failures, please?

**Comment by Baaaaam** [comment]
I find a way to have all test successful...
it is a bit messy right now... but I try to make it as simple as possible...
